react-frontend/src/containers/App/index.js

```diff
@@ -13,8 +13,9 @@ function App() {
   return (
     <div className="App">
       <Router>
-        <CovidBanner />
         <NavBar />
+        {/* CovidBanner is moved down so the skip navigation is first tag on the page*/}
+        <CovidBanner />
```

Comment:
Not sure this is what we want. On gov.bc.ca, the COVID banner is presented before the header: 

![tab-order](https://user-images.githubusercontent.com/16946297/94957797-42184800-04a3-11eb-979e-1b839d69dc80.gif)

Reply:
I have reverted this so the banner comes second.